modules/plugin/jdbc/jdbc-spatialite/src/test/java/org/geotools/data/spatialite/SpatiaLiteDataStoreAPITestSetup.java

```diff
@@ -28,7 +30,16 @@
     protected SpatiaLiteDataStoreAPITestSetup() {
         super(new SpatiaLiteTestSetup());
     }
-    
+
+    protected void setUpData() throws Exception {
+        //kill all the data
+        super.setUpData();
+        try {
+            dropDataTypesTable();
```

Comment:
Perhaps I am a bit slow, I thought these tests do clean up activities (like dropDataTypesTable) in the teardown method, and in the setup method prior to setting up the data (incase there was a previous failed run).